apps/frontend/src/modules/users/libs/validation-schemas/validation-schemas.ts

```diff
@@ -1 +1,5 @@
-export { userSignInValidationSchema, userSignUpValidationSchema } from "shared";
+export {
+	type SignUpFormValidationSchema,
```

Comment:
you should import types from shared folder in types.ts file and import it from there